src/scss/_objects.content-card.scss

```diff
@@ -0,0 +1,41 @@
+.content-card {
+  margin: halve($inuit-base-spacing-unit) 0;
```

Comment:
I didn't want to add a Responsive watcher just to remove the horizontal padding (for the row-oriented/tile-like ContentCards) in mobile, so I added a media query setting the horizontal padding to 0px. Let me know if you think we should add the Responsive watcher to update the pad props in the JS. Thanks!